articles/create-a-node-cli/index.md

```diff
@@ -0,0 +1,257 @@
+# Creating Command Line Interfaces using Node.js
+
+The command line or the terminal is a powerful tool used by a lot of developers. It allows developers to interact with the operating system using commands. These commands are passed as input strings of text. A lot of tools used by developers have a terminal CLI or command line interface to go with the product. Using the command line is easy and direct hence why a lot of people prefer using it.
```

Comment:
Consider splitting up your larger paragraphs into smaller ones. This makes your article easy to read, especially for mobile users.